initramfs/init.sh

```diff
@@ -34,7 +34,7 @@ restart()
 	sleep 5s
 	reboot
 	modprobe sunxi_wdt || true
-	watchdog -T 1 -t 60 -F /dev/watchdog || true
+	watchdog -T 1 -t 60 -F "/dev/watchdog" || true
```

Comment:
while the change is good, better would be to put it into a variable :)
as for the || true, which is my fault, I think it should have said:
```sh
if [ -w "${WATCHDOG_DEV}" ]; then
    watchdog -T 1 -t 60 -F "${WATCHDOG_DEV}"
fi
```
instead.